crates/rolldown/src/stages/generate_stage/compute_cross_chunk_links.rs

```diff
@@ -64,10 +64,7 @@ impl<'a> GenerateStage<'a> {
       &mut index_imports_from_other_chunks,
     );
 
-    let index_sorted_cross_chunk_imports = index_cross_chunk_imports
-      .into_iter()
-      // FIXME: Extra traversing. This is a workaround due to `par_bridge` doesn't ensure order https://github.com/rayon-rs/rayon/issues/551#issuecomment-882069261
```

Comment:
Should keep this. What this PR does is still to be considered as a workaround.

Reply:
why this is a workaround?

Reply:
According to https://github.com/rayon-rs/rayon/issues/669#issuecomment-507814077, the ordering now is preserved without another traverse

Reply:
https://github.com/rolldown/rolldown/pull/2016/files#r1721029588

Reply:
It's not about rayon here. IMO, this should be solved by the `oxc_index` to have a native support of rayon.

Reply:
Then that should be TODO: wait oxc_index support `IntoParIterator` rather than `Fixme:`, since the bug is fixed, no overhead needed.

Reply:
Yeah.